publish.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+rm -rf ./output
+git clone https://github.com/OpenRA/openra.github.io.git output
```

Comment:
For me it is the other way round. I have to enter my SSH password all the time. HTTPS credentials are cached.